resolwe/flow/executors/local/run.py

```diff
@@ -21,7 +21,6 @@ def __init__(self, *args, **kwargs):
         """Initialize attributes."""
         super(FlowExecutor, self).__init__(*args, **kwargs)
 
-        self.processes = {}
```

Comment:
@kostko was there any good reason for this dict instead of just using `self.proc`?

Reply:
No idea, I didn't add this?